types/k6/tsconfig.json

```diff
@@ -42,6 +44,8 @@
         "test/data.ts",
         "test/encoding.ts",
         "test/execution.ts",
+        "test/experimental/browser.ts",
```

Comment:
Also here.

Reply:
Sorry for the rebase mishap, fixing it 👍🏻